Library/Formula/libiodbc.rb

```diff
@@ -0,0 +1,22 @@
+require 'formula'
+
+class Libiodbc < Formula
+  homepage 'http://www.iodbc.org/dataspace/iodbc/wiki/iODBC/'
```

Comment:
Also, could you add a short description above the homepage using `desc "…"`? Thanks!